lib/plugins/babelRuntimePathResolver.js

```diff
@@ -2,11 +2,12 @@
 // in OBT's node_modules directory and not the module's
 'use strict';
 
-const path = require('path');
 const falafel = require('falafel');
 
 const prefix = /babel-runtime/;
-const rootDirectory = path.resolve(__dirname, '../../node_modules');
+
+// HACK: Can't require.resolve babel-runtime so use babel-core instead
+const rootDirectory = require.resolve('babel-core').replace(/(.*node_modules).*/, '$1');
```

Comment:
@AlbertoElias just run those snippets in dev tools and you'll see it works :innocent:

this is what the regex is doing:-
![screen shot 2015-10-29 at 07 52 38](https://cloud.githubusercontent.com/assets/825088/10812848/0f4a7b04-7e12-11e5-894e-c7e83c799ecc.png)